charts/opensearch/README.md

```diff
@@ -96,7 +96,8 @@ helm uninstall my-release
 | `service.transportPortName`        | The name of the transport port within the service                                                                                                                                                                                                         | `transport`                                     |
 | `service.type`                     | OpenSearch [Service Types][]                                                                                                                                                                                                                           | `ClusterIP`                                     |
 | `sidecarResources`                 | Allows you to set the [resources][] for the sidecar containers in the StatefulSet                                                                                                                                                                         | {}                                              |
-| `sysctlInitContainer`              | Allows you to disable the `sysctlInitContainer` if you are setting sysctl vm.max_map_count` with another method                                                                                                                                        | `enabled: true`                                 |
+| `sysctl`              | Allows you to enable the `sysctl` to set sysctl vm.max_map_count through `securityContext`. This requires privilege.                                                                                                                                         | `enabled: false`                                 |
```

Comment:
I don't think "requires privilege" is enough detail here because the situation is quite complex. Could you add a bit more detail about how this won't work unless you add an argument to the kubelet, and maybe link through to https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/?

Reply:
We only need `privileged: true` for `sysctlInitContainer` (that is being pushed via this PR) and not for general `sysctl` and this gets added via  `corev1.PodSecurityContext`, so should we mention this line `This requires privilege` ?
@smlx @amirgo1

Reply:
@prudhvigodithi I agree with you and think `.Values.sysctl` need to be removed along with `PodSecurityPolicy` as it shouldn't work at all.
vm.max_map_count sysctl is not namespaced, so it cannot be configured through securityContext.

As explained here:
https://github.com/opensearch-project/helm-charts/issues/87#issuecomment-942738258

@smlx - what do you think?

Reply:
I think that removing `.Values.sysctl` is unrelated to the changes in this PR. It belongs in another PR, and it's a breaking change so would need a major version bump.

Reply:
I took this out of the documentation (it wasn't there before).